src/_includes/partials/token_table.njk

```diff
@@ -13,7 +13,7 @@
     {%- endcomment -%}
     {% assign concatTokens = token | split: " " %}
     {% for concatToken in concatTokens %}
-        {%- assign filteredTokens = concatToken | split: "." | dig: tokens['Tokens'] %}
+        {%- assign filteredTokens = concatToken | split: "." | dig: tokens.tokens['Tokens'] %}
```

Comment:
Do you think it would be clearer to rename the tokens.json into figma.json? so that we access it through `tokens.figma['Tokens']`

Reply:
Yes I like that!